specification/network/resource-manager/readme.md

```diff
@@ -695,6 +696,9 @@ directive:
   - suppress: RequiredPropertiesMissingInResourceModel
     from: virtualNetworkGateway.json
     reason: name, id and type properties are inherited from the upper level
+  - suppress: RequiredPropertiesMissingInResourceModel

```

Comment:
Was this approved? Please forward the email thread offline

Reply:
Were the APIs reviewed? Yes, forwarded offline.

Reply:
Not the API, the suppressions requested here, did not see anything in the email thread regarding this

Reply:
Why does the suppression need ARM review? The properties are included via the Resource class (note the $allOf in the InterfaceEndpoint definition). The warning from the Linter is n/a, hence the suppression.

Reply:
Hmm.. looks like this has been approved for all other Network resources. Won't block on it,